support-frontend/assets/pages/supporter-plus-landing/components/paymentTsAndCs.tsx

```diff
@@ -121,11 +124,15 @@ export function TsAndCsFooterLinks({
 
 	const terms = amountIsAboveThreshold
 		? termsSupporterPlus('Terms and Conditions')
+		: inAdLite
+		? termsGuardianAdLite('Terms')
 		: termsContributions;
```

Comment:
Interesting, I've never used [conditional chains](https://developer.mozilla.org/en-US/docs/Web/JavaScript/Reference/Operators/Conditional_operator#conditional_chains) in a ternary before!

Reply:
Is that ok to do if its simple or better to breakout into multiple line?

Reply:
I _thought_ about suggesting an alternative here but actually I think it's quite elegant! I think the alternatives (`if/else if` or a `switch`) would require defining a `let` variable and then setting. It's quite nice here just assigning the result straight to a `const`.